jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -240,6 +248,39 @@ node {
         isGARelease = hasNewGARelease(componentRelease, latestRelease, productName)
         println "[INFO] latestRelease:${latestRelease}, componentRelease:${componentRelease}, isGARelease:${isGARelease}"
         currentBuild.description = "latest: ${latestRelease}\ncurrent: ${componentRelease}"
+
+        if (releaseTagVar == "") {
+          productVersion = latestRelease
+        }
+
+        println "[INFO] product version: ${productVersion}"
+    }
+
+    stage('Fetch Component Product Version') {
+        when(productVersion == "") {
```

Comment:
Why do this if productVersion is blank? Maybe if we skipped this if the productVersionVar param was blank instead?